### PR TITLE
[trantor] Update to 1.0.0

### PR DESCRIPTION
**Pull request by an-tao**

### CODE REVIEW

**Review by NancyLi1013 — APPROVED** [review]

**Review by ras0219 — APPROVED** [review]

File: ports/trantor/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: trantor
-Version: v1.0.0-rc16
+Version: v1.0.0
```
Comment (ras0219) [inline ports/trantor/CONTROL:2]:
```suggestion
Version: 1.0.0
```

In vcpkg, we remove preceeding `v`'s from library versions.

**Pull request merged**